docs/src/main/asciidoc/firestore.adoc

```diff
@@ -159,6 +160,43 @@ Repositories allow you to define custom Query Methods (detailed in the following
 
 NOTE: Custom queries with `@Query` annotation are not supported since there is no query language in Cloud Firestore
 
+
+=== Firestore Operations & Template
+
+`FirestoreOperations` and its implementation, `FirestoreTemplate`, provides the Template pattern familiar to Spring developers.
+
+Using the auto-configuration provided by Spring Data Cloud Firestore, your Spring application context will contain a fully configured `FirestoreTemplate` object that you can autowire in your application:
```

Comment:
What not recommend autowiring `FirestoreOperations`?

Reply:
switched to `FirestoreOperations`